src/index.js

```diff
@@ -39,6 +46,55 @@ class Offline {
     this.exitCode = 0;
     this.provider = 'aws';
     this.start = this.start.bind(this);
+    this.coverageDataOutputPath = process.env.SLS_OFFLINE_COVERAGE_DATA_OUTPUT_PATH;
+    this.shouldOutputCoverage = typeof this.coverageDataOutputPath !== 'undefined';
+
+    if (this.shouldOutputCoverage) {
+      debugLog(
+        `Code coverage will be written to ${this.coverageDataOutputPath}`
+      );
+      /* eslint-disable new-cap */
+      this.coverageQueue = new pQueue({ concurrency: 1 });
+      /* eslint-enable new-cap */
```

Comment:
No need to re-enable if you implement other suggestion. (for applying bulk suggestions):
```suggestion
      /* eslint-enable new-cap */
```